catroid/src/main/java/org/catrobat/catroid/content/bricks/GoToBrick.java

```diff
@@ -152,19 +148,19 @@ public View getPrototypeView(Context context) {
 	public List<SequenceAction> addActionToSequence(Sprite sprite, SequenceAction sequence) {
 		sequence.addAction(sprite.getActionFactory().createGoToAction(sprite, destinationSprite, spinnerSelection));
 
-		return null;
+		return Collections.emptyList();
```

Comment:
Hehe, yeah! However

>  "Ensure no-op code refactoring is done separately from functional changes"

or better, create atomic commits. The improvement should be in it's own commit. It's just easier to review. Otherwise if someone looks at it later, he will think this change is necessary for the fix.